geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DescribeRegionCommand.java

```diff
@@ -359,7 +272,7 @@ private boolean writeFixedPartitionAttributesToTable(TabularResultData table,
         fpaBuilder.append(fpa.getNumBuckets());
 
         if (!isTypeAdded) {
-          type = attributeType;
+          type = "";
```

Comment:
I agree with the removal of a constant parameter, but in this case, we're not distinguishing between `type = "";` here vs `type = blank; // = ""` a few lines below. 
 These could be removed altogether, with `""` passed directly to the `writeAttributeToTable` call at the end of this method.